patch-package
--- a/node_modules/next-server/dist/lib/utils.d.ts
+++ b/node_modules/next-server/dist/lib/utils.d.ts
@@ -75,6 +75,10 @@ export interface NextPageContext {
      * `String` of the actual path including query.
      */
     asPath?: string;
+
+    // @s-h-a-d-o-w
+    // Necessary for with-redux-store
+    [key: string]: any;
 }
 export declare type AppContextType<R extends BaseRouter = BaseRouter> = {
     Component: NextComponentType<NextPageContext>;
